src/internal_modules/roc_audio/packetizer.cpp

```diff
@@ -131,7 +132,13 @@ void Packetizer::end_packet_() {
         pad_packet_();
     }
 
-    writer_.write(packet_);
+    const status::StatusCode code = writer_.write(packet_);
+    if (code != status::StatusOK) {
+        // TODO: handle returned status code (gh-183).
+        roc_panic_if_msg(code != status::StatusNoData,
+                         "packetizer: failed to write packet: status=%s",
+                         status::code_to_str(code));
```

Comment:
Here and in similar places: I think writer can never return StatusNoData (what would it mean?). So here we need unconditional panic (for now, later we will unconditionally forward code I think).